streams/shared/src/main/scala/zio/stream/ZTransducer.scala

```diff
@@ -727,6 +727,23 @@ object ZTransducer {
       }
     }
 
+  /**
+   * Creates a transducer that limits byte chunks to the given size.
+   */
+  def throttleBytes(limit: Int): ZTransducer[Any, Nothing, Byte, Chunk[Byte]] =
+    throttleChunks[Byte](limit)
```

Comment:
We can drop this one as it's a monomorphic version of the other variant.

Reply:
Removed